detect_test.go

```diff
@@ -0,0 +1,57 @@
+package enable
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/paketo-buildpacks/packit/v2"
+	"github.com/paketo-buildpacks/packit/v2/scribe"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+)
+
+func testDetect(t *testing.T, context spec.G, it spec.S) {
```

Comment:
I am not sure, but shouldn't we check for the build plan requirements in this test somehow? So if a `package.json` file exists, we also require node_modules for example.

Reply:
will check this our

Reply:
modified the test to check if we set the plan and if is correct len when package.json is set